src/main/java/carracing/domain/Game.java

```diff
@@ -0,0 +1,55 @@
+package carracing.domain;
+
+import carracing.view.OutputView;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Game {
+
+    private int carCount = 0;
+    private int tryCount = 0;
+    private List<Car> cars = null;
+
+    public Game(int carCount, int tryCount){
+        inputValid(carCount, tryCount);
+
+        this.tryCount = tryCount;
+        this.cars = this.generateCars(carCount);
+
+    }
+
+    private void inputValid(int carCount, int tryCount) {
+        if( carCount == 0 || tryCount == 0) throw new IllegalArgumentException("입력값을 확인해주세요.");
```

Comment:
```suggestion
        if (carCount == 0 || tryCount == 0) {
            throw new IllegalArgumentException("입력값을 확인해주세요.");
        }
```
중괄호는 생략하지 않는것이 컨벤션 입니다 !